charts/jellyseerr-chart/values.yaml

```diff
@@ -16,6 +16,25 @@ fullnameOverride: ""
 strategy:
   type: Recreate
 
+# Liveness / Readiness / Startup Probes
+probes:
+  # -- Configure liveness probe (set to null to disable)
+  livenessProbe:
+    httpGet:
+      path: /
+      port: http
+    initialDelaySeconds: 60
+  # -- Configure readiness probe (set to null to disable)
+  readinessProbe:
+    enabled: true
```

Comment:
There is no enabled field inside [probe](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#probe-v1-core)